services/api/src/routes/auth.js

```diff
@@ -32,11 +32,12 @@ router
         ctx.throw(400, 'A user with that email already exists');
       }
 
-      const authTokenId = generateTokenId();
-      const user = await User.create({
-        authTokenId,
+      const user = new User({
         ...ctx.request.body,
       });
+      const { token, payload } = createAuthToken(user);
```

Comment:
I would kind of want to see the creation of the auth token delegated to a single method, but I'm guessing there's a reason this isn't possible?

Reply:
I think that could be possible, was split out like that to split apart token creation and adding the token to the user object ... but they can be combine too